fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -768,29 +776,39 @@ private void handleQueryStmt(List<PlanFragment> fragments, List<ScanNode> scanNo
         }
     }
 
-    private void handleAnalyzeStmt() throws Exception {
+    private void handleAnalyzeStmt() {
         AnalyzeStmt analyzeStmt = (AnalyzeStmt) parsedStmt;
-        StatisticExecutor statisticExecutor = new StatisticExecutor();
         Database db = MetaUtils.getDatabase(context, analyzeStmt.getTableName());
-        Table table = MetaUtils.getTable(context, analyzeStmt.getTableName());
+        OlapTable table = (OlapTable) MetaUtils.getTable(context, analyzeStmt.getTableName());
 
-        AnalyzeJob job = new AnalyzeJob(db.getId(), table.getId(), analyzeStmt.getColumnNames(),
+        AnalyzeJob analyzeJob = new AnalyzeJob(db.getId(), table.getId(), analyzeStmt.getColumnNames(),
                 analyzeStmt.isSample() ? Constants.AnalyzeType.SAMPLE : Constants.AnalyzeType.FULL,
                 Constants.ScheduleType.ONCE,
                 analyzeStmt.getProperties(),
-                Constants.ScheduleStatus.FINISH,
-                LocalDateTime.now());
+                Constants.ScheduleStatus.RUNNING,
```

Comment:
The new job should be PENDING?

Reply:
No.This position is to collect statistics manually (synchronously) and is already in the running state. Penning is mainly suitable for automatic (asynchronous) collection of statistics